Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Builder image to compile the website
-FROM ubuntu:focal as builder
+FROM ubuntu as builder
```

Comment:
I am assuming that you are looking to upgrade to the LTS version. 

However, to avoid unexpected upgrades in the future, I would personally specify that we now want to use "jammy".

Reply:
However, I can see that your change makes sense if you prefer to ensure that we are always using the latest LTS version. Looking to hearing your reasoning for this change!

Reply:
It's a long overdue revert of a temporary change I made (#978).